doc/INSTALL

```diff
@@ -1,244 +1,215 @@
+      libqt5x11extras5-dev qtbase5-dev qtmultimedia5-dev
+      
+  Note: You can install QT5.x and QT4.x libraries side by side.
+  You may want to only install a preferred Qt version instead of both at same time.
+  Then to try a different QT version, uninstall one before installing the other, e.g.
+
+    $ sudo apt-get remove --purge <package> <package> <package> etc.
+    $ sudo apt-get install <package> <package> <package> etc.
+      
+-------------------------------------------------------------------------------
+  3. Configuring the environment:
+-------------------------------------------------------------------------------
 
-    This document contains the procedures to compile and install
-    the version 4.2 of KVIrc.
+  If your distribution is sane installing the development package
+  should set up everything correctly. However in the event it does not
+  you may need to export the paths for your libraries before compiling.
 
-###############################################################################
-# 0. Table of contents
-###############################################################################
+  You will know that something is wrong when you run CMake,
+  as it will complain paths are not found e.g. $QTDIR
+  If configuration terminates successfully you can skin # 3., else
+  you might need to set it with the command:
 
-    1. Introduction
-    2. Minimal requirements
-    3. Configuring the environment
-    4. Interactive mode
-    5. Compiling
-    6. Compiling on MacOS X
-    7. Compiling on Win32
-    8. Hacker-level installation
-    9. Creating a KVIrc package
+    $ export QTDIR="your Qt dir"
 
-###############################################################################
-# 1. Introduction
-###############################################################################
+  For x86 e.g.
 
-    This document contains the procedures to compile and install
-    the version 4.2 of KVIrc.
+    $ export QTDIR=/usr/lib/qt4
 
-    If you have found this document in a binary distribution then
-    KVIrc has been probably already installed by your favorite package
-    manager and maybe something is not working as expected. In this case
-    this document and the accompanying FAQ can help you in guessing
-    what's wrong.
+  For x86_64 e.g.
+  
+    $ export QTDIR=/usr/lib/qt4 
+ 
+  If you have multiple versions of Qtx.x installed and want to use a specific version,
+  exporting QTDIR won't suffice, since the CMake module searching for Qtx.x
+  uses qmake itself to determine the Qt4 library dir.
+  You can set your $PATH variable to have CMake using your preferred Qtx.x version:
 
-    If you have found this document in a source distribution or you
-    have downloaded it by using SVN then well...
-    this is a standard INSTALL file :)
+    $ PATH=/opt/qtx.x/bin/:$PATH cmake ..
+    
+  Note: The trailing dots are required or CMake will NOT create the right
+  environment.
```

Comment:
They are important *if* you do `mkdir build; cd build` in source tree.
Otherwise, it'll do something weird (most probably, just not find CMakeLists.txt)